scss/widgets/base/_gridBase.scss

```diff
@@ -82,7 +82,7 @@
       }
     }
 
-    &.dx-#{$widget-name}-borders {
+    &.dx-#{$widget-name}-borders, &> .dx-#{$widget-name}-borders {
```

Comment:
Why two locations of .dx-#{$widget-name}-borders class here? May be we can remove &.dx-#{$widget-name}-borders and it was wrong?